content/block_docs.md

```diff
@@ -35,16 +36,24 @@ hash: String
 ```
 
 ## Coinbase
-The proposer of the block is identified by the first transaction in the `transaction_list`.
-This transaction is called the *coinbase* and will get awarded the block mining reward for their work.
+The proposer of the block is identified by the source of the first transaction in `transaction_list`.
+This transaction is called the *coinbase*, and its source will get awarded by the block mining reward for their work.
+The amount of the reward is determined by `block_reward` field of [`/config`](/config).
 
-> Place one of your own transactions as the first item in `transaction_list`
+> Place one of your own transactions as the first item in `transaction_list`.
+> Otherwise, someone else will receive the reward!
```

Comment:
otherwise it doesn't work since owner of coinbase transaction is also authenticating with the JWT

Reply:
Right, I forgot about that